SwiftWisdom/Rx/Rx+Extensions.swift

```diff
@@ -25,45 +25,61 @@ infix operator <-> : Binding
 infix operator <- : Binding
 infix operator >>> : Disposing
 
-public func <- <T: ObserverType, O: ObservableType>(observer: T, observable: O) -> Disposable where T.E == O.E {
+public func <- <T: ObserverType, O: ObservableType>(observer: T, observable: O) -> Disposable where T.Element == O.Element {
     return observable.observeOn(MainScheduler.instance).bind(to: observer)
 }
 
-public func <- <T: ObserverType, O: ObservableType>(observer: T, observable: O) -> Disposable where T.E == O.E? {
+public func <- <T: ObserverType, O: ObservableType>(observer: T, observable: O) -> Disposable where T.Element == O.Element? {
     return observable.observeOn(MainScheduler.instance).bind(to: observer)
 }
 
-public func <- <T: ObserverType, O>(observer: T, variable: Variable<O>) -> Disposable where T.E == O {
-    return observer <- variable.asObservable()
+public func <- <T: ObserverType, O>(observer: T, variable: O) -> Disposable where T.Element == O {
+    return observer <- BehaviorRelay(value: variable)
 }
 
-public func <- <T: ObserverType, O>(observer: T, variable: Variable<O>) -> Disposable where T.E == O? {
-    return observer <- variable.asObservable()
+public func <- <T: ObserverType, O>(observer: T, variable: O) -> Disposable where T.Element == O? {
+    return observer <- BehaviorRelay(value: variable)
 }
 
-public func <- <T, O: ObservableType>(variable: Variable<T>, observable: O) -> Disposable where O.E == T {
-    return observable.bind(to: variable)
-}
+//public func <- <T, O: ObservableType>(variable: T, observable: O) -> Disposable where O.Element == T {
```

Comment:
Is there a reason to comment out this code rather than just deleting it?

Reply:
Just my paranoia - but the commented out code is deleted now